packages/zowe-explorer/CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 All notable changes to the "vscode-extension-for-zowe" extension will be documented in this file.
 
+## `1.13.0`
+
+- Added the monorepo landing Readme that contains the high-level overview of the repository folders such as `packages` folder, instructions on how to contribute to the project and links to Medium articles providing additional useful information about Zowe Explorer and Zowe [#1199](https://github.com/zowe/vscode-extension-for-zowe/pull/1199). Thanks @IgorCATech
+- Fixed the issue that prevented the list of recently opened files from being displayed upon request. You can access a list of recently opened files by pressing the Ctrl+Alt+R or Command+Alt+R key combination [#1208](https://github.com/zowe/vscode-extension-for-zowe/pull/#1208). Thanks @jellypuno
+- Fixed the issue that caused the USS tree to collapse after renaming a USS directory [#650](https://github.com/zowe/vscode-extension-for-zowe/issues/650). Thanks @lauren-li
```

Comment:
Looks like this issue (650) will not go in for v1.13.0 release. And I think we need to replace it with issue 992 -  the file picker command has been fixed for this release.